src/Controller/Component/RequestHandlerComponent.php

```diff
@@ -300,6 +300,7 @@ public function beforeRedirect(Event $event, $url, Response $response)
                 'REQUEST_METHOD' => 'GET'
             ]
         ]));
+        $response->statusCode(200);
         $response->send();
         $response->stop();
     }
```

Comment:
By "Let's do that" I meant, return the response object with the right code instad of `send()` and `stop()`